umm-spec-lib/src/cmr/umm_spec/spatial_conversion.clj

```diff
@@ -51,7 +39,8 @@
   [vertical-spatial-domains]
   (->> vertical-spatial-domains
        (map #(xml-parse/fields-from % :Type :Value))
-       drop-invalid-vertical-spatial-domains))
+       drop-invalid-vertical-spatial-domains
+       (map #(update % :Type (fn [x] (string/replace x "_" " "))))))
```

Comment:
So I think you could do this line before `drop-invalid-vertical-spatial-domains` and then you wouldn't have to do the `(string/replace...)` on line 24.